Pages/Index.cshtml

```diff
@@ -51,8 +51,9 @@
 
                 <!-- Search -->
                 <div class="col-12 col-md-5">
-                    <label class="form-label">Search</label>
+                    <label class="form-label" for="search-box">Search</label>
                     <input asp-for="Query.Q"
+                            id="Search-box"
```

Comment:
Careful here, the case (uppercase) is different between the label and the input id.  The browser will likely figure it out, but best to stay consistent.
Also, do you have a best practice citation for this?